games-board/gtkatlantic/gtkatlantic-0.6.3.ebuild

```diff
@@ -0,0 +1,34 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+
+inherit autotools xdg-utils
+
+DESCRIPTION="Monopoly-like game that works with the monopd server"
+HOMEPAGE="http://gtkatlantic.gradator.net/"
+SRC_URI="http://download.tuxfamily.org/gtkatlantic/downloads/v0.6/${P}.tar.bz2"
```

Comment:
[$(ver_cut)?](https://mgorny.pl/articles/the-ultimate-guide-to-eapi-7.html#version-manipulation-and-comparison-commands)